build.sbt

```diff
@@ -143,7 +143,7 @@ lazy val macros = project
   .settings(Settings.macros)
 
 lazy val vectortile = project
-  .dependsOn(vector)
+  .dependsOn(vector, layer)
```

Comment:
btw should it still depend on the layer project?

Reply:
Ah, I added that for the test, but you're right that we don't have to depend on it. I'll remove it now :+1: